app/models/package_manager/psc_package.rb

```diff
@@ -0,0 +1,35 @@
+module PackageManager
+  class PscPackage < Base
+    HAS_VERSIONS = false
+    HAS_DEPENDENCIES = false
+    BIBLIOTHECARY_PLANNED = true
+    URL = 'https://github.com/purescript/psc-package'
+    COLOR = '#1D222D'
+
+    def self.project_names
+      get('https://raw.githubusercontent.com/purescript/package-sets/master/packages.json').keys
+    end
+
+    def self.project(name)
+      project = get('https://raw.githubusercontent.com/purescript/package-sets/master/packages.json')[name]
+      project['name'] = name
```

Comment:
This is done so that we'll have access to the `name` field in `#mapping` and other methods, as the JSON returned otherwise does not include it.

Reply:
👍